### PR TITLE
Update Richie's profile.

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

- [x] Updated CHANGELOG.md
- [x] Add PR label
  - Update Richie's profle.

### CODE REVIEW

File: tests/unit/specs/src/layouts/AccessWalletLayout/components/SoftwareModal/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!doctype html>
```
Comment (gamalielhere) [inline tests/unit/specs/src/layouts/AccessWalletLayout/components/SoftwareModal/index.html]: What's this file for?

File: src/layouts/TeamLayout/TeamLayout.vue [hunk 2]
```diff
@@ -73,7 +73,7 @@ export default {
         {
           name: 'Duke & Richie Wolf',
           position: 'Blockchain Specialist',
-          otherInfo: '#MEWForce',
+          //otherInfo: '#MEWForce',
```
Comment (kvhnuke) [inline src/layouts/TeamLayout/TeamLayout.vue]: comment :(

File: tests/unit/specs/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (gamalielhere) [inline tests/unit/specs/package.json]: What's this file for?

Comment (kvhnuke) [reply tests/unit/specs/package.json]: can you fix this?

**Pull request merged**